lib/docblock-python.js

```diff
@@ -155,10 +156,27 @@ function activate(state) {
   // Events subscribed to in atom's system can be easily cleaned up with a
   // CompositeDisposable
   this.subscriptions = new CompositeDisposable();
+  this.stylechooser = new StyleChooserView(state.StyleChooserViewState);
+
+  // Add view provider
+  atom.views.addViewProvider(() => {
+    return new StyleChooserView(state.StyleChooserViewState).element;
+  });
```

Comment:
This view provider is the thing that is conflicting with other packages. According to the [docs](https://flight-manual.atom.io/api/v1.53.0/ViewRegistry/#instance-addViewProvider) you should provide a constructor as the first argument to prevent this from getting called by every call to `atom.views.getView()`

```diff
- atom.views.addViewProvider(() => {
-   return new StyleChooserView(state.StyleChooserViewState).element;
+ atom.views.addViewProvider(StyleChooserView, (model) => {
+   return model.element;
  });
```